roles.json

```
@@ -295,107 +274,87 @@
     {
         "role": "Sub-package maintainer",
         "url": "Subpackage_maintainer",
-        "role-head": "Sub-package maintainer (1 per core package sub-package)",
+        "role-head": "Sub-package maintainer (at least one per core package sub-package)",
         "sub-roles": [
             {
                 "role": "astropy.constants",
-                "lead": ["David Shupe"],
-                "deputy": ["Marten van Kerkwijk"]
+                "people": ["David Shupe", "Marten van Kerkwijk"]
             },
             {
                 "role": "astropy.convolution",
-                "lead": ["Adam Ginsburg"],
-                "deputy": ["Axel Donath", "Larry Bradley"]
+                "people": ["Adam Ginsburg", "Axel Donath", "Larry Bradley"]
             },
             {
                 "role": "astropy.coordinates",
-                "lead": ["Erik Tollerud"],
-                "deputy": ["Stuart Littlefair", "Adrian Price-Whelan", "Juan Luis Cano Rodr\u00edguez"]
+                "people": ["Erik Tollerud", "Stuart Littlefair", "Adrian Price-Whelan", "Juan Luis Cano Rodr\u00edguez", "Marten van Kerkwijk"]
             },
             {
                 "role": "astropy.cosmology",
-                "lead": ["Unfilled"],
-                "deputy": ["Unfilled"]
+                "people": ["Unfilled"]
             },
             {
                 "role": "astropy.io.ascii",
-                "lead": ["Tom Aldcroft"],
-                "deputy": ["Hans Moritz G\u00fcnther", "Derek Homeier"]
+                "people": ["Tom Aldcroft", "Hans Moritz G\u00fcnther", "Derek Homeier"]
             },
             {
                 "role": "astropy.io.fits",
-                "lead": ["Unfilled"],
-                "deputy": ["Simon Conseil", "Michael Seifert"]
+                "people": ["Simon Conseil", "Michael Seifert"]
             },
             {
                 "role": "astropy.io.misc",
-                "lead": ["Unfilled"],
-                "deputy": ["Matteo Bachetti", "Ed Slavich"]
+                "people": ["Matteo Bachetti", "Ed Slavich"]
             },
             {
                 "role": "astropy.io.votable",
-                "lead": ["Pey Lian Lim"],
-                "deputy": ["Thomas Boch", "Tom Donaldson", "Pey Lian Lim"]
+                "people": ["Pey Lian Lim", "Thomas Boch", "Tom Donaldson", "Pey Lian Lim"]
```

No need to list me twice. 😅 
```suggestion
                "people": ["Thomas Boch", "Tom Donaldson", "Pey Lian Lim"]
```